projects/common.py

```diff
@@ -27,11 +27,17 @@ def install_packages(packages, proxy=''):
         sudo('yum -y install {0}'.format(' '.join(packages)), pty=False)
 
 
+def update_packages(packages, proxy=''):
+    with shell_env(http_proxy=proxy, ftp_proxy=proxy):
+        sudo('yum -y update {0}'.format(' '.join(packages)), pty=False)
```

Comment:
Ah we should also move the "proxy" argument from being an argument to being a fabric `env` variable.